js-webapp/README.md

```diff
@@ -188,7 +224,7 @@ following HTML:
             const newtonraphson = new NewtonRaphson(tolerance);
             const root = newtonraphson.solve(initial_guess);
             // Write root to tag with answer as identifier
-            document.getElementById("answer").innerHTML = root.toFixed(2);
+            document.getElementById("answer").innerHTML = root.toFixed(4);
```

Comment:
The cli has a precision of 2. Why do we have 4 here?

Reply:
Doh, you are correct, I wanted to compromise and go to 4, but changed my mind halfway.